relay/README.md

```diff
@@ -41,10 +41,6 @@ The circuit relay is both a tunneled transport and a mounted swarm protocol. The
      +-----------------------------------------+     +-----------------------------------------+
 ```
 
-**Notes for the reader:**
-
-- We're using the `/p2p` multiaddr protocol instead of `/ipfs` in this document. `/ipfs` is currently the canonical way of addressing a libp2p or IPFS node, but given the growing non-IPFS usage of libp2p, we'll migrate to using `/p2p`.
```

Comment:
I think it's still true that we're migrating from `/ipfs` to `/p2p` but the migration is in no way done. Someone reading the spec and then looking at the implementations might be confused unless we keep this note.

Reply:
maybe say "we _are_ migrating".